src/main/kotlin/app/CodeLongevity.kt

```diff
@@ -96,7 +96,7 @@ class CodeLongevity(repoPath: String, tailRev: String) {
             if (author.getEmailAddress() != email) {
                 continue
             }
-            line.printme()
+            println(line.toString())
```

Comment:
In prod logger should be used for debug output.

Reply:
I'll remove those on integration stage I think